crates/bevy_ecs/macros/src/lib.rs

```diff
@@ -118,13 +118,13 @@ pub fn derive_bundle(input: TokenStream) -> TokenStream {
     {
         if *is_bundle {
             field_type_infos.push(quote! {
-                type_info.extend(#field_type::type_info());
+                type_info.extend(<#field_type>::type_info());
```

Comment:
If you're going to wrap the type in angle brackets, would it be worth specifying `Bundle` as well to avoid conflicts if the type (for some reason) has an associated `type_info` function defined on itself?

```suggestion
                type_info.extend(<#field_type as #ecs_path::bundle::Bundle>::type_info());
```

Reply:
Makes sense!